packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,420 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/list';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import collectionsDocs from 'docs:@react-types/shared/src/collections.d.ts';
+import selectionDocs from 'docs:@react-stately/selection';
+import statelyDocs from 'docs:@react-stately/list';
+import focusDocs from 'docs:@react-aria/focus';
+import checkboxDocs from 'docs:@react-aria/checkbox';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+import packageData from '@react-aria/list/package.json';
+import Anatomy from './anatomy.svg';
+
+---
+category: Collections
+keywords: [list, aria, grid]
+---
+
+# useList
+
+<PageDescription>{docs.exports.useList.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useList', 'useListItem', 'useListSelectionCheckbox']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/patterns/grid/'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useList} links={docs.links} />
+<FunctionAPI function={docs.exports.useListItem} links={docs.links} />
+<FunctionAPI function={docs.exports.useListSelectionCheckbox} links={docs.links} />
+
+## Features
+
+A list can be built using [&lt;ul&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) or [&lt;ol&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol) HTML elements, but is very limited in functionality especially when it comes to user interactions.
+HTML lists are meant for static content, rather than lists with rich interactions like focusable elements within rows, keyboard navigation, row selection, etc.
+`useList` helps achieve accessible and interactive list components that can be styled as needed.
+
+* Exposed to assistive technology as a `grid` using ARIA
+* Keyboard navigation between rows and in-rows focusable elements via the arrow keys
+* Single, multiple, or no row selection via mouse, touch, or keyboard interactions
+* Support for disabled rows, which cannot be selected
+* Optional support for checkboxes in each row for selection
+* Support for both `toggle` and `replace` selection behaviors
+* Support for row actions via double click, <Keyboard>Enter</Keyboard> key, or tapping
+* Long press to enter selection mode on touch when there is both selection and row actions
+* Async loading and infinite scrolling
+* Typeahead to allow focusing rows by typing text
+* Automatic scrolling support during keyboard navigation
+* Labeling support for accessibility
+* Ensures that selections are announced using an ARIA live region
+* Virtualized scrolling support for performance with large lists
+
+## Anatomy
+
+<Anatomy />
```

Comment:
The selected row background color is kinda hard to distinguish in dark mode, can you change that color to whatever we are currently using for our ListView? I think I used `var(--spectrum-alias-highlight-selected)`

Reply:
good catch, fixed